justfile

```diff
@@ -30,7 +30,8 @@ check-formatting:
 # Cross compiles embedded-graphics for a target
 build-target target *args:
     cargo build --target {{target}} {{args}}
-    cargo build --target {{target}} --all-features {{args}}
+    cargo build --target {{target}} --features nalgebra {{args}}
+    cargo build --target {{target}} --features fixed {{args}}
```

Comment:
If we go with this solution: Should we test both features in one build or use separate ones? I think the total build time has increased, but this would test the features independent of each other.

Reply:
I'm on the fence. Rerunning the build leaves it at 2m 10s which is still pretty good.

Cargo features are supposed to be additive, right, so *should* be safe to compile together. We also know that `nalgebra` and `fixed` are unlikely to conflict with each other, so I'd be fine merging these two back together if you wanted to.

Reply:
Sure, they should be sage to compile together, but compiling them separately makes sure they don't depend on each other.

But I think building it once is enough. I've added an `all_features` variable to the justfile and use that in all instances that used `--all-features` before. This should minimize the build time for `build-all` because it doesn't need to build criterion anymore.